pkg/sass-parser/package.json

```diff
@@ -37,12 +37,14 @@
   "devDependencies": {
     "@types/jest": "^29.5.12",
     "copyfiles": "^2.4.1",
+    "eslint-plugin-import": "^2.30.0",
     "expect": "^29.7.0",
     "gts": "^5.0.0",
```

Comment:
is it needed to still have a direct dependency on ègts` or is the transitive dependency enough (`sass-analysis` will use its own `gts` dependency)